storybook/stories/core/components/Form/ValidatableFrom.story.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <oxd-text tag="h5">Add Job Category</oxd-text>
+
+  <oxd-divider />
+
+  <oxd-form @submit="checkForm" @submitValid="getFormValues" ref="form">
+    <oxd-form-row>
+      <oxd-form-group class="orangehrm-bottom-space">
+        <oxd-input-field
+            label="Job Category Name"
+            v-model="name"
+            :rules="rules.name"
```

Comment:
How we add multiple rules . min/max chars ?

Reply:
Sure, Here we reference the `rules` object. under the key `name` there are mutliple rules, required and max chars.

```
rules {
        name: [
          v => (!!v && v.trim() !== "") || "Required",
          v => (v && v.length <= 50) || "Should be less than 50 characters"
        ],
       ...
}
```
And if we want min chars added we may define it as well.
`v => (v && v.length >= 10) || "Should more than 10 characters"`